english/candyland/eating/stream/stream.md

```diff
@@ -0,0 +1,5 @@
+You have arrived at the stream. You drink deeply and quench your thirst. A fascinating tower sits ablazed in the distance. What do you want to do now?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.